app/refresh-ingestion.sh

```diff
@@ -161,9 +159,18 @@ EOF
 
 echo_stats(){
     local DATASET_ID="$1"
+    echo "------ $DATASET_ID ----------------"
+    if [ "$DATASET_ID" == "imagine_la" ]; then
+        ls -ld src/ingestion/imagine_la/scrape/pages
+        echo "HTML files scraped: "
+        ls src/ingestion/imagine_la/scrape/pages | wc -l
+    else
+        grep -E 'log_count|item_scraped_count|request_depth|downloader/|httpcache/' "logs/${DATASET_ID}-1scrape.log"
+    fi
```

Comment:
A bit of refactoring for `echo_stats` so that it can be called at the end of `all` ingestions to make it easier to copy and paste into slack.